bundler.py

```diff
@@ -71,7 +71,17 @@ def setup_4337_contracts(config: L2Config):
             config.deployer_key = priv_key
         else:
             priv_key = config.deployer_key
-        lib.run("set private key", f"echo PRIVATE_KEY={priv_key} > account-abstraction/.env")
+        lib.run("set deployment key", f"echo PRIVATE_KEY={priv_key} > account-abstraction/.env")
+        # set private key for paymaster
+        if config.paymaster_key is None:
+            paymaster_key = input("Enter private key for paymaster signer: ")
+            config.paymaster_key = paymaster_key
+        else:
+            paymaster_key = config.paymaster_key
+        lib.run(
+            "set paymaster key",
+            f"echo PAYMASTER_PRIVATE_KEY={paymaster_key} >> account-abstraction/.env"
+        )
```

Comment:
What about getting rid of the variable altogether, in favor of just using `config.paymaster_key` everywhere?

Reply:
Certainly very good point, have done that now.